Backend/src/modules/users/users.route.ts

```diff
@@ -30,11 +28,22 @@ const controller = new Controller();
  * @param {CreateUserBody} request.body.required
  * @return {User} 201 - user created
  */
-users.post(
-  '/create',
-  verifyAuthToken,
-  RequestValidator.validate(CreateUserDto),
-  controller.createUser
+/** Login */
+// auth login
+users.get('/auth/login', controller.authLogin);
+// auth with google
+users.get(
+  '/auth/google',
+  passport.authenticate('google', {
+    scope: ['profile', 'email'],
+  })
 );
-
+// callback route for google to redirect
+users.get(
+  '/auth/google/redirect',
+  passport.authenticate('google'),
+  controller.authRedirectGoogle
```

Comment:
do we need this line?

Reply:
yes, It is very important in the whole process

Reply:
but you already have `passport.authenticate('google'),`
so why you need to link the controller too?
from passportjs documentation, it seemed to be not important to link additional controller.

Reply:
It will be used later to redirect to profile api.